pywavefront/material.py

```diff
@@ -95,7 +97,7 @@ def pad_light(self, values):
 
     def set_alpha(self, alpha):
         """Set alpha/last value on all four lighting attributes."""
-        alpha = float(alpha)
+        self.transparency = alpha
```

Comment:
Is there a reason to remove the enforced `float` here?

If this is restored, the two `float` can be removed (see L187, and L192).

Reply:
My reasoning: The Material api should take floats as input. It's the parsers responsibility to provide the right types.